CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.36.2
```

Comment:
Can you rebase your change to the newest dev branch? This does not belong to your changes.

Reply:
yeah i rebased everything hopefully it's good now